src/style.css

```
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+body {
+  margin: 20px 8px 8px;
+}
+
+.searchBar_input {
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-size: 1em;
+  font-weight: 400;
```

You don't have such imports of the font.
maybe you wanted wright 
```suggestion
  font-weight: 700;
```